### PR TITLE
#1 day1 solution

### PR DESCRIPTION
**Pull request by kxa14**
Solved Day1 puzzle but failed to pass one of the tests.

### CODE REVIEW

File: src/main/scala/advent/solutions/Day1.scala [hunk 1]
```diff
@@ -1,46 +1,55 @@
...
-    * @return The fuel required to launch the module, plus the fuel required to launch that fuel
-    */
-  def totalFuel(mass: Int): Int = ???
+   *
+   * @param mass The mass of the module
+   * @return The fuel required to launch the module, plus the fuel required to launch that fuel
+   */
+  def totalFuel(mass: Int): Int = {
+
+    @tailrec
```
Comment (zainab-ali) [inline src/main/scala/advent/solutions/Day1.scala:37]: Awesome use of `@tailrec` :smile:

File: src/main/scala/advent/solutions/Day1.scala [hunk 2]
```diff
@@ -1,46 +1,55 @@
 package advent.solutions
 
+import scala.annotation.tailrec
+
 /** Day 1: The Tyranny of the Rocket Equation
```
Comment (zainab-ali) [inline src/main/scala/advent/solutions/Day1.scala:5]: I've added a formatter (`scalafmt`) so we'll not need to worry about formatting changes.  Could you fetch the most recent changes and then run `sbt scalafmtAll`?

**Review by zainab-ali — CHANGES_REQUESTED** [review]
Thanks @KXA194 !  Could you fetch the latest changes from master?  There was a typo in the test that should be fixed there.

**Comment by kxa14** [comment]
I've re-submitted a new pull request. Sorry for screwing up with git. 😄

**Comment by zainab-ali** [comment]
Closing as solved by #8

**Pull request closed without merging**